docs/getting-started/wallets.md

```diff
@@ -292,14 +292,14 @@ wallet = bt.wallet( name = 'my_coldkey', hotkey = 'my_first_hotkey', path = '~/p
 ```python
 create_coldkey_from_uri(self, uri:str, use_password: bool = True, overwrite:bool = False) -> 'Wallet'
 ```
-Creates coldkey from suri string, optionally encrypts it with the user's inputed password.
+Creates coldkey from uri string, optionally encrypts it with user entered password.
```

Comment:
This is a commented out section, marked off for removal or reuse somewhere later. I will not review this section of the PR.

Reply:
Ok, no difference. Left these with their fixes